bin/citrea/tests/test_helpers.rs

```diff
@@ -37,6 +37,7 @@ pub async fn start_rollup(
     db_path: Option<&str>,
     min_soft_confirmations_per_commitment: u64,
     include_tx_body: bool,
+    test_mode: Option<bool>,
```

Comment:
None and Some(false) are equal. So there is no need to make test_mode `Option<bool>`, make it bool with default=false if it's missing in toml config.